tool/src/cpp2/ty.rs

```diff
@@ -593,9 +593,7 @@ inline {ty_name} {ty_name}::FromFFI({ctype} c_struct) {{
     ) -> Option<Cow<'a, str>> {
         match *result_ty {
             ReturnFallability::Infallible(None) => None,
-            ReturnFallability::Infallible(Some(ReturnType::Writeable)) => {
-                Some("/* TODO: Writeable conversion */".into())
-            }
+            ReturnFallability::Infallible(Some(ReturnType::Writeable)) => Some("output".into()),
```

Comment:
suggestion: comment mentioning where the `output` comes from (it was created earlier in `writeable_prefix`)